ontology/yaml/resources/HVAC/entity_types/FCU.yaml

```diff
@@ -19,6 +19,17 @@
 # This file defines fan-coil unit types. Valid types must include a fan and a coil of some type.
 # They must also include some control scheme (e.g. ZTC, DSP, etc.).
 
+FCU_SS_CO2M_ZTC_STC_RTM:
+  description: "AC unit with heating and cooling (zone temp control)"
+  is_canonical: true
+  implements:
+  - FCU
+  - SS
```

Comment:
This is ambiguous - is this SS control for the fan? The cooling/heating element?

Reply:
Yes the SS is for the entire unit

Reply:
I would expect there to be separate SS for the fan (DFSS, etc) and the conditioning elements (DXZTC, HTZTC, etc)

Reply:
Okay thanks a million - we only have a single run command and status from the bms points but I will check them. Thank you